Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml

```
@@ -251,6 +251,13 @@ script:
       - 'false'
       required: false
       secret: false
+    - default: false
+      defaultValue: '100'
+      description: 'The Minimum confidence required of IPs to retrieve. Default is 100.'
```

```suggestion
      description: The Minimum confidence required for the retrieved IPs.
```
No need to note the default version, the UI will do it automatically.